pkg/gardenlet/apis/config/types.go

```diff
@@ -403,6 +407,22 @@ type SNIIngress struct {
 	Labels map[string]string
 }
 
+// ETCDConfig contains ETCD related configs
+type ETCDConfig struct {
+	// EnableBackupCompaction enables automatic compaction of etcd backups
+	// Defaults to false
+	// +optional
```

Comment:
Drop `// +optional` tags and `json` tags from internal APIs

Reply:
Done